python-modules/cis_change_service/cis_change_service/idp.py

```diff
@@ -0,0 +1,131 @@
+import json
+import logging
+
+from functools import wraps
+from flask import request
+from flask import _request_ctx_stack
+from six.moves.urllib.request import urlopen
+from jose import jwt
+
+from cis_change_service import get_config
+from cis_change_service.exceptions import AuthError
+
+logger = logging.getLogger(__name__)
+
+CONFIG = get_config()
+
+AUTH0_DOMAIN = CONFIG('auth0_domain', namespace='change_service', default='auth-dev.mozilla.auth0.com')
+API_IDENTIFIER = CONFIG('api_identifier', namespace='change_service', default='https://change.sso.allizom.org')
```

Comment:
maybe this should be api.sso.allizom.org ;-)

Reply:
these are going to end up being different backend flask apps and I'm not sure at present how to unify them.  So change in my mind was separate from api.

Reply:
change.api.sso.allizom.org?
not a big deal either way